Autocomplete/predefined-items-and-recent-searches/src/recent-searches.js

```diff
@@ -0,0 +1,29 @@
+import { createRecentSearchesPlugin } from '@algolia/autocomplete-plugin-recent-searches';
+const orgRecentSearches = createRecentSearchesPlugin({
```

Comment:
`org` confused me at first—could stand for "organization". `original` would be more explicit.